CMakeLists.txt

```diff
@@ -38,17 +38,17 @@ set (HMI "web" CACHE STRING "HMI type")
 option(HMI2 "Use Qt HMI" OFF)
 option(EXTENDED_MEDIA_MODE "Turn on and off extended Madia Manager features relates to PulseAudio A2DP and GStreamer" OFF)
 option(BUILD_SHARED_LIBS "Build all libraries as shared (if ON) or static (if OFF)" OFF)
-option(BUILD_BT_SUPPORT "Bluetooth support" ON)
+option(BUILD_BT_SUPPORT "Bluetooth support" OFF)
```

Comment:
IS it temporary?
Please, add comment in case of any temporary solutions (with ticket mentioning)